src/cms/templates/push_notifications/push_notification_form.html

```diff
@@ -4,10 +4,26 @@
 {% load static %}
 {% load widget_tweaks %}
 {% load push_notification_filters %}
+{% load page_filters %}
+{% load rules %}
+{% load compress %}
+
+<script>
+    function switch_language(language) {
+        var languages = [{% for lang in languages %}'{{ lang.id }}', {% endfor %}];
+        console.log(languages);
+        languages.forEach(function(language){
+            u("#tab-"+language).addClass("hidden");
+            u("#li-"+language).removeClass("z-10");
+        });
+        u("#tab-"+language).removeClass("hidden");
+        u("#li-"+language).addClass("z-10");
+}
+</script>
```

Comment:
I'd propose to outsource any js code which is more than 2 lines to additional js files which are then included in the template instead of inline code.

Reply:
Done